README.md

```diff
@@ -1,19 +1,21 @@
-This repository contains a set of Helm charts that will be made available as quick starts to all Teams in the Otomi developer catalog.
+# Otomi Catalog quick starts
 
-The following charts are available:
+This repository contains a set of Helm charts that will be made available as quick starts to all Teams in the Otomi Catalog.
+
+The following charts are added to the Catalog:
 
 ## otomi-quickstart-k8s-deployment
 
-The `otomi-quickstart-k8s-deployment` chart can be used to create a Kubernetes `Deployment`. Optionally a `HPA` and a `Service Monitor` can be created.
+The `otomi-quickstart-k8s-deployment` Helm chart can be used to create a Kubernetes `Deployment` (to deploy a single image), a `Service` and a `ServiceAccount`. Optionally a `HorizontalPodAutoscaler`, a Prometheus `ServiceMonitor` and a `Configmap` can be created.
 
 ## otomi-quickstart-k8s-deployment-otel
 
-The `otomi-quickstart-k8s-deployment-otel` chart can be used to create a Kubernetes `Deployment` with Open Telemetry `instrumentation`.
+The `otomi-quickstart-k8s-deployment-otel` Helm chart can be used to create a Kubernetes `Deployment` (to deploy a single image), a `Service`, a `ServiceAccount`, an `OpenTelemetryCollector` and an `Instrumentation`. Optionally a `HorizontalPodAutoscaler`, a Prometheus `ServiceMonitor` and a `Configmap` can be created.
 
 ## otomi-quickstart-k8s-deployments-canary
 
-The `otomi-quickstart-k8s-deployments-canary` chart can be used to create 2 Kubernetes `Deployments` to deploy 2 versions of an image and use the `Traffic Control` feature in Otomi to do canary or blue/green deployments.
+The `otomi-quickstart-k8s-deployment-canary` Helm chart can be used to create 2 Kubernetes `Deployments` (to deploy 2 versions of an image), a `Service` and a `ServiceAccount`. Optionally a `HorizontalPodAutoscaler`, a Prometheus `ServiceMonitor` and a `Configmap` (for each version) can be created.
```

Comment:
Should we change the folder name?
If we change the folder name (without s in deployments), we also need to change the rbac.yaml file content.

Reply:
fixed